src/main.cpp

```diff
@@ -509,8 +511,13 @@ class Konstructs: public nanogui::Screen {
     void handle_belt(const string &str) {
         uint32_t column, size, type, health;
         if(sscanf(str.c_str(), ",%u,%u,%u,%u",
-                  &column, &size, &type, &health) != 4)
-            throw std::runtime_error(str);
+            &column, &size, &type, &health)!=4)
+        {
+            std::string what="handle_belt expected ,%u,%u,%u,%u got "+str;
+
+            cout<<what;
+//            throw std::runtime_error(what);
+        }
```

Comment:
The server was spitting out only 3 values for this and considering I have no clue the issue I remarked it for the moment as in order to catch other crashes I had all exceptions turned on (didn't mean to keep it in the check-in)